src/main/resources/locales/en/accessibility-statement.json

```diff
@@ -1,44 +1,45 @@
 {
   "title": "Accessibility statement for the Court and Tribunal Hearing Information Service",
   "section1": {
-    "text1": "The Court and Tribunal Hearing Information Service provides a publishing platform to enable information provided by HMCTS to be shared, displayed and updated as appropriate. It will enable lists, outcomes, judgments and information to be displayed to court users, consistent with jurisdictional procedures and business rules. The service supports HMCTS’ commitment to open justice and the provision of transparent and consistent court/tribunal information across all jurisdictions.",
-    "text2": "This accessibility statement applies to content published on ",
-    "text3": " It is based on an accessibility audit for carried out by the Digital Accessibility Centre (DAC) user / technical team on 1 March 2022.",
+    "text1": "This accessibility statement applies to content published on ",
+    "text2": "The Court and Tribunal Hearings Service provides a publishing platform to enable information, such as hearing lists, provided by HMCTS to be shared, displayed and updated as appropriate. It will enable information to be displayed to court users, consistent with jurisdictional procedures and business rules. The service supports HMCTS’ commitment to open justice and the provision of transparent and consistent court and tribunal information across jurisdictions",
     "text4": "This website is run by HM Courts and Tribunals. We want as many people as possible to be able to use this website, so we have designed it to be accessible. For example, that means you should be able to:",
     "text5": "We’ve also made the website text as simple as possible to understand.",
-    "text6": "has advice on making your device easier to use if you have a disability.",
+    "text6": "(opens in a new window) has advice on making your device easier to use if you have a disability.",
```

Comment:
The link doesn't open in a new window atm

Reply:
should be fixed now!